packages/account-v2/src/modules/FinancialAssessmentFields/FinancialAssessmentFields.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useMemo } from 'react';
+import { useFormikContext } from 'formik';
+import { useIsEuRegion } from '@deriv/api-v2';
+import { FormDropDownField } from '../../components/FormFields';
+import {
+    accountTurnoverList,
+    educationLevelList,
+    EMPLOYMENT_VALUES,
+    employmentIndustryList,
+    employmentStatusList,
+    estimatedWorthList,
+    incomeSourceList,
+    netIncomeList,
+    sourceOfWealthList,
+} from '../../constants/financialInformationList';
+import { filterOccupationList, shouldHideOccupation } from '../../utils/financialAssessmentUtils';
+import type { TFinancialAssessmentFormValues } from '../types';
+import { financialAssessmentValidations } from './validations';
+
+export const FinancialAssessmentFields = () => {
+    const { isEUCountry } = useIsEuRegion();
+
+    const formik = useFormikContext<TFinancialAssessmentFormValues>();
+
+    if (!formik) {
+        throw new Error('FinancialAssessmentFields must be used within a Formik component');
+    }
+
+    const { setFieldValue, values } = formik;
+    const { employmentStatus, occupation } = values;
+
+    const hideOccupation = shouldHideOccupation(employmentStatus || '');
+
+    const filteredOccupationList = useMemo(() => filterOccupationList(employmentStatus || ''), [employmentStatus]);
+
+    useEffect(() => {
+        if (employmentStatus === EMPLOYMENT_VALUES.employed && occupation === EMPLOYMENT_VALUES.unemployed) {
```

Comment:
this condition is a bit confusing, like how `employment state` is `employed` AND `occupation` can be `unemployed` ??

Reply:
I think here we are clearing the occupation field so the user should reselect it in this condition.